nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -148,6 +148,24 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 - (Neo)Vim can not be configured with `configure.pathogen` anymore to reduce maintainance burden.
 Use `configure.packages` instead.
 
+- The WordPress module now stores themes and plugins in a directory name without appending the current version. This avoids loosing settings when upgrading themes and plugins as some of them use the directory name to store the settings in the database. This needs manual intervention in case of using such a broken plugin or theme which is rare.
+  ```bash
+  nix-shell -p wp-cli
+  cat /etc/httpd/httpd.conf  | grep "\-wordpress-"
+  cd /nix/store/xxx-wordpress-xxx/share/wordpress
+  sudo -u wordpress wp maintenance-mode activate
```

Comment:
We can't automate this?

Reply:
I don't think so as there *may* be strings that should not be replaced or that need further action. On the other side the theme that *was* broken in the past doesn't seem to be broken any more and I don't know of any other themes or plugins that are broken (though I didn't use many). So I would expect that most people don't need to take any actions and the search there should return empty results.